cedar-policy/src/api.rs

```diff
@@ -2281,14 +2309,19 @@ pub enum TemplatePrincipalConstraint {
     /// Must be equal to the given EntityUid.
     /// If [`None`], then it is a template slot.
     Eq(Option<EntityUid>),
+    /// Must be the given EntityTypeName.
+    Is(EntityTypeName),
+    /// Must be the given EntityTypeName, and `in` the EntityUID
+    /// If the EntityUID is [`None`], then it is a template slot.
```

Comment:
```suggestion
    /// Must be the given EntityTypeName, and `in` the EntityUID.
    /// If the EntityUID is [`None`], then it is a template slot.
```
Markdown will run these together as a single sentence, so we either need a period (as I suggested here), or a blank line of just `///` to force a Markdown paragraph break